runtime/doc/various.txt

```diff
@@ -454,10 +454,12 @@ g8			Print the hex values of the bytes used in the
 							*:verbose-cmd*
 When 'verbose' is non-zero, listing the value of a Vim option or a key map or
 an abbreviation or a user-defined function or a command or a highlight group
-or an autocommand will also display where it was last defined.  If it was
-defined manually then there will be no "Last set" message.  When it was
-defined while executing a function, user command or autocommand, the script in
-which it was defined is reported.
+or an autocommand will also display where it was last defined. If they were
+defined through lua api they will only be located if verbose level was
```

Comment:
- always capitalize lua in docs and messages, unless it is referring to a filename or symbol.
- Acronyms like "API" are all caps unless... they are filenames or symbols. (btw: "id" is not an acronym).
- "the Lua API" is generally not mentioned. It's just "Lua".

 
```suggestion
defined in Lua they will only be located if 'verbose' is set.
```